keyboards/sol/keymaps/danielhklein/keymap.c

```diff
@@ -0,0 +1,360 @@
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN, KC_RGHT, KC_INS,  KC_END,  \
+      _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MPLY, KC_MUTE, KC_VOLD, KC_VOLU, KC_MRWD, KC_MFFD, \
+      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______, XXXXXXX, _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+                        XXXXXXX, XXXXXXX,          XXXXXXX, _______, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX \
+      ),
+
+  /* ADJ
+   * ,------------------------------------------------.  ,------------------------------------------------.
+   * |RESET |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  |      |   =  |   /  |   *  |   -  |      |      |
+   * |------+------+------+------+------+------|------|  |------|------+------+------+------+------+------|
+   * |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |  |      |   7  |   8  |   9  |   +  |      |      |
+   * |------+------+------+------+------+------|------|  |------|------+------+------+------+------+------|
+   * |      | HUI  | VAI  | SAI  |      |      |COLEMK|  |      |   4  |   5  |   6  | Enter|      |      |
+   * |------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+   * |      | HUD  | VAD  | SAD  |      |      |QWERTY|  |      |   1  |   2  |   3  | Space|      |      |
+   * |------+------+------+------+------+------+------|  |------+------+------+------+------+------+------|
+   * |      |RGBRST|RGBTOG|RGBMOD|RGBSMD|      |      |  |      |      |   0  |   .  | Bspc |      |      |
+   * `------+------+------+------+------+------+------|  |------+------+------+------+------+------+------'
+   *                                    |      |      |  |      |      |
+   *                                    `-------------'  `-------------'
+   */
+
+  [_ADJ] =  LAYOUT( \
+      RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,    KC_F5,   KC_F6,   XXXXXXX, KC_PEQL, KC_PSLS, KC_PAST, KC_PMNS, XXXXXXX, XXXXXXX, \
+      XXXXXXX, KC_F7,   KC_F8,   KC_F9,   KC_F10,   KC_F11,  KC_F12,  XXXXXXX, KC_P7,   KC_P8,   KC_P9,   KC_PPLS, XXXXXXX, XXXXXXX, \
+      XXXXXXX, RGB_SAD, RGB_VAI, RGB_SAI, XXXXXXX,  XXXXXXX, COLEMAK, XXXXXXX, KC_P4,   KC_P5,   KC_P6,   KC_PENT, XXXXXXX, XXXXXXX, \
+      XXXXXXX, RGB_HUD, RGB_VAD, RGB_HUI, XXXXXXX,  XXXXXXX, QWERTY,  XXXXXXX, KC_P1,   KC_P2,   KC_P3,   KC_SPC,  XXXXXXX, XXXXXXX, \
+      XXXXXXX, RGBRST,  RGB_TOG, RGB_MOD, RGB_SMOD, XXXXXXX, XXXXXXX, _______, XXXXXXX, KC_P0,   KC_PDOT, KC_BSPC, XXXXXXX, XXXXXXX, \
+                        XXXXXXX, XXXXXXX,           XXXXXXX, XXXXXXX, _______, XXXXXXX,          XXXXXXX, XXXXXXX \
+      )
+};
+
+// define variables for reactive RGB
+bool TOG_STATUS = false;
+int RGB_current_mode;
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer